.travis.yml

```diff
@@ -1,13 +1,15 @@
 ---
+sudo: false
 language: ruby
 
+notifications:
+  email: false
+
 rvm:
   - 2.1.9
```

Comment:
Which ruby versions should we support? 2.1.9 is what puppet4 uses and puppet5 is currently on 2.4.1. Should we support both? others?